client/src/main/java/io/pravega/client/stream/impl/EventStreamReaderImpl.java

```diff
@@ -152,6 +152,7 @@ private EventRead<Type> readNextEventInternal(long timeoutMillis) throws ReaderN
 
             EventSegmentReader segmentReader = orderer.nextSegment(readers);
             if (segmentReader == null) {
+                log.debug("Reader {} have no segment to read. No of available segments = {}.", groupState.getReaderId(), readers.size());
```

Comment:
Please, fix the typo in the message: `have` -> `has`.
Also, please change `No` by `Number`, as `No` can be easily interpreted as "NO". Thanks.

Reply:
Done.